flutter/test/backend/loadgen_info_test.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:flutter_test/flutter_test.dart';
```

Comment:
It's good to have test added 👍

Reply:
Please also update 
https://github.com/mlcommons/mobile_app_open/blob/6b36eb3835159821f5f862d18b4b779e7d0c3f59/flutter/flutter.mk#L179-L180

to have this new test target.

Reply:
I don't think it should be part of this PR, which is about updating loadgen.
I have created a separate issue for this: https://github.com/mlcommons/mobile_app_open/issues/485